### PR TITLE
Feature/storage/new quick query features

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/src/QuickQueryExtensions.cs [hunk 1]
```diff
@@ -52,9 +52,18 @@ internal static class QuickQueryExtensions
                     RecordSeparator = jsonTextConfiguration.RecordSeparator?.ToString(CultureInfo.InvariantCulture)
                 };
             }
+            else if (textConfiguration.GetType() == typeof(BlobQueryArrowOptions))
```
Comment (gapra-msft) [inline sdk/storage/Azure.Storage.Blobs/src/QuickQueryExtensions.cs:55]: Do you have a separate input and output converter?

Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/src/QuickQueryExtensions.cs:57]: Fixed!

**Review by gapra-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/tests/BlobQuickQueryTests.cs [hunk 2]
```diff
@@ -456,6 +456,81 @@ public async Task QueryAsync_IfTags_Failed()
...
...AAAEAAAAUAAAAEAAUAAgABgAHAAwAAAAQABAAAAAAAAEHJAAAABQAAAAEAAAAAAAAAAgADAAEAAgACAAAAAQAAAACAAAABAAAAE5hbWUAAAAAAAAAAP////9wAAAAEAAAAAAACgAOAAYABQAIAAoAAAAAAwMAEAAAAAAACgAMAAAABAAIAAoAAAAwAAAABAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAP////+IAAAAFAAAAAAAAAAMABYABgAFAAgADAAMAAAAAAMDABgAAAAAAgAAAAAAAAAACgAYAAwABAAIAAoAAAA8AAAAEAAAACAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAABAAAAIAAAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAAkAEAAAAAAAAAAAAAAAAAAJABAAAAAAAAAAAAAAAAAACQAQAAAAAAAAAAAAAAAAAA", Convert.ToBase64String(memoryStream.ToArray()));
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Blobs/tests/BlobQuickQueryTests.cs:495]: Should this be playback only ?

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/tests/BlobQuickQueryTests.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs/tests/BlobQuickQueryTests.cs:495]: I think it should be ok live, but I will switch it to playback online if we have trouble when I start working on the live tests.

**Pull request merged**